Azure/azure-cli
# [App Service] `az webapp create`: Add new parameter `--acr-use-identity` to configure pull image from ACR using MSI when creating a container web app

**Related command**
`az webapp create`

**Description**
Add new parameter `--acr-use-identity` to configure pull image from ACR using MSI when creating a container web app. It allows user to update siteconfig acrUseManagedIdentityCreds while creating webapp, this can significantly reduce steps user needs to have to create a container webapp which pulls image from ACR using MSI.

**Testing Guide**
1. Create webapps by running `az webapp create` command with and without `--acr-use-identity` to make sure webapp can be created successfully with new parameter also changes don't break webapp creation without this new parameter. And check the config of new created webapp by using `az webapp config show` to make sure siteconfig acrUseManagedIdentityCreds is updated as expected.
2. Create test methods test_webapp_hyperv_acr_use_identity and test_webapp_linux_acr_use_identity to test `az webapp create` with this parameter

**History Notes**


[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.
